examples/tools/system/package_manager.rst

```diff
@@ -0,0 +1,229 @@
+.. _examples_tools_system_package_manager:
+
+Wrapping system requirements in a Conan package
+===============================================
+
+Conan can manage system packages, allowing you to install platform-specific dependencies easily.
+This is useful when you need to install platform-specific system packages.
+For example, you may need to install a package that provides a specific driver or graphics library that only works on a specific platform.
+
+Conan provides a way to install system packages using the :ref:`system package manager<conan_tools_system_package_manager>` tool.
+
+In this example, we are going to explore the steps needed to package a system library and what is needed to consume it in a Conan package.
+For this illustration, we are going to create a Conan package to wrap the system `ncurses <https://invisible-island.net/ncurses/>`_
```

Comment:
``For this illustration`` => In this example

Reply:
Done, please, see the commit 7bcd1abd